app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class='signup-div'>
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :surname %><br />
+    <%= f.text_field :surname %>
+  </div>
+
```

Comment:
- [ ] It would be better if our registration view matches up with the wireframe. Here we are missing the`photo` and `bio` fields. Here is a screenshot of the wireframe

![Screenshot from 2022-11-10 14-11-19](https://user-images.githubusercontent.com/46388113/201114286-1bcce563-d9e9-4855-ac94-1f535ee81f11.png)
![Screenshot from 2022-11-10 14-12-17](https://user-images.githubusercontent.com/46388113/201114497-2b4e6687-d05a-4c82-af38-cc64b578f26e.png)